### PR TITLE
NT: zona pectinata of basilar membrane of cochlea

### LINKED ISSUE #1357: UBERON:0002820 needs to be split into two classes
`UBERON:0002820` seems to have been copied from NIFGA which contained a copy and paste error that falsely equated `UMLS:C1184862` -> `Zona arcuata of basilar membrane of cochlea` and `UMLS:C1184863` -> `Zona pectinata of basilar membrane of cochlea`. Given the current equivalent class mappings to NIFGA I recommend that `UMLS:C1184863` -> `Zona pectinata of basilar membrane of cochlea` be created as a new class and point to `BIRNLEX:2529` consistent with the [recent change to NIFGA](https://github.com/SciCrunch/NIF-Ontology/commit/5a46231839a92ea44e17105958edb867e97f90f0). http://purl.obolibrary.org/obo/uberon/bridge/uberon-bridge-to-nifstd.owl should be updated as well.

### PR DESCRIPTION
**Pull request by cmungall**
also added grouping class for zonas pectinata and arcuate

Placed UMLS xrefs in correct place.

Hopefully places nlx IDs also in correct place

Fixes #1357

### CODE REVIEW

**Comment by cmungall** [comment]
@tgbugs - this look ok? I opted not to formally split (which would mean obsoleting the zona arcuata) and budded off a new term instead; open to argument that original concept was sufficiently ambiguous that obsolete required

also sent you invite to repo, so I can tag you as reviewer

File: uberon_edit.obo [hunk 1]
```diff
@@ -101992,22 +101992,46 @@ xref: UMLS:C0458765 {source="NIFSTD:birnlex_2555"}
...
+id: UBERON:0036249
+name: zona pectinata of basilar membrane of cochlea
+def: "The acellular membrane that forms the outer two thirds of the basilar membrane of the conchlea." [http://medical-dictionary.thefreedictionary.com/zona+pectinata, https://github.com/obophenotype/uberon/issues/1357]
+synonym: "zona pectinata" BROAD []
+synonym: "pectinate zone" BROAD []
+synonym: "pectinate zone of organ of corti" EXACT []
+synonym: "pectinate zone of basilar membrane" EXACT []
+is_a: UBERON:0005764 ! acellular membrane
+relationship: part_of UBERON:0002226 ! basilar membrane of cochlea
+xref: NIFSTD:birnlex_2529 ! Zona arcuata of basilar membrane of cochlea
```
Comment (tgbugs) [inline uberon_edit.obo]: The latest version fixes this so can use `NIFSTD:birnlex_2529 ! Zona pectinata of basilar membrane of cochlea`

**Comment by tgbugs** [comment]
Ah, yes, nomenclature mismatch on the `split` budding preferred (I should know better having worked on yeast...).

**Comment by cmungall** [comment]
thanks!

btw failing checks is a false positive, difference in shells on travis...

**Pull request merged**